lib/splitclient-rb/clients/split_client.rb

```diff
@@ -279,5 +320,19 @@ def treatment(
 
       parsed_treatment(multiple, treatment_data)
     end
+
+    def properties_size(properties)
+      properties.inject(0) { |accum, (key, value)|
+        case value
+          when Numeric then accum += value.to_s.length
```

Comment:
for consistency between the other sdks and simplicity, we are only considering the length of the string

Reply:
Updated